services/app-api/emailer/templates.test.ts

```diff
@@ -314,19 +326,18 @@ describe('Email templates', () => {
         })
 
      })
-     describe('CMS unlock email', () =>{
+    describe('CMS unlock email', () =>{
+        const unlockData = {
+            submissionName: 'MCR-VA-CCCPLUS-0001',
```

Comment:
this is very nice. I don’t think we’ve done stuff like this anywhere and I bet it would help. The only risk here is that that unlock data is shared across all these test runs, if any test were to modify it, that might make a different test fail which could be confusing. We’ve solved that problem elsewhere by writing functions that return a new object every time they are called, you can see that pattern in apolloHelpers.tsx. In this case, this is probably fine since we don’t expect any of these functions to do anything but read this, but it’s something to be aware of.